string state
float64 angle1
float64 angle2
float64 angle3
float64 angle4
float64 arm_palm
float64 palm_finger
float64 finger_finger_tip